components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gene_suggestions.xml

```diff
@@ -121,16 +121,67 @@
 $xwiki.jsx.use('PhenoTips.PanelsExport')##
 $xwiki.jsx.use('PhenoTips.PanelsDisplay')##
 $xwiki.ssx.use('PhenoTips.PanelsDisplay')##
+$xwiki.jsfx.use('js/xwiki/table/livetable.js', true)##
+$xwiki.ssfx.use('js/xwiki/table/livetable.css', true)##
   (% class="instant-search gene-panels-search-box" %)(((
-    === $services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.matchingGenes')  {{html wiki=false clean=false}}&lt;span class="buttonwrapper"&gt;&lt;a href="#" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" id="genePanelsExport" class="button gene-panels-button csv-export-button"&gt;&lt;span class="fa fa-fw fa-download"&gt;&lt;/span&gt; $services.localization.render('phenotips.UIXField.geneSuggestions.download')&lt;/a&gt;&lt;/span&gt;{{/html}}{{icon name="question-circle" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" cssClass="xHelpButton"/}} ===
+    === $escapetool.xml($services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.matchingGenes'))  {{html wiki=false clean=false}}&lt;span class="buttonwrapper"&gt;&lt;a href="#" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" id="genePanelsExport" class="button gene-panels-button csv-export-button"&gt;&lt;span class="fa fa-fw fa-download"&gt;&lt;/span&gt; $services.localization.render('phenotips.UIXField.geneSuggestions.download')&lt;/a&gt;&lt;/span&gt;{{/html}}{{icon name="question-circle" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" cssClass="xHelpButton"/}} ===
```

Comment:
`$escapetool.xml` only makes sense when inside a `{{html}}` macro, otherwise it's not needed. You should remove it from here and two lines down.